VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 | ------- | ----------- | --------------- | -------------------- |
+| 3.0.0   | 4.0.0       | 5.0.0           | 3.2.4                |
```

Comment:
iOS is 4.9.0 right?

Reply:
Yup, you're right! https://github.com/RevenueCat/cordova-plugin-purchases/pull/153